cpu.go

```diff
@@ -1,16 +1,23 @@
 package main
 
 import (
+	"fmt"
 	"github.com/shirou/gopsutil/cpu"
+	"time"
 )
 
 //func GetCpuTemperature() {
 //	//Implement function to get the current temperature of CPU
 //}
 //
-//func GetCpuUsage() {
-//	//Implement function to get current usage of CPU
-//}
+func GetCpuUsage() {
+	cpuPercent, err := cpu.Percent(time.Second, false)
+	if err != nil {
+		StandardPrinter(ErrorRedColor, "Could not retrieve CPU usage details.")
+	}
+	usedPercent := fmt.Sprintf("%f", cpuPercent[0])
```

Comment:
try formatting this to two decimal places

Reply:
ok,  done